### ISSUE TITLE
Printing has no text on any but the first page

### ISSUE THREAD

**Issue by kannes** [kentonv/etherpad-lite#46]
When printing a Etherpad using its print button text only appears on the first page of the output (I used a PDF printer). Author colors for the lines do appear in the margins of subsequent pages so there is "something".

This bug happens in Sandstorm-hosted Etherpads. I hope this is the right repo to report bugs with that, if not, please accept my excuses.

**Comment by kentonv** [comment]
Yep, this is the right place to report it. The bad news is I probably won't be able to look at this any time soon. :( Etherpad is pretty bad at things like export and printing, and fixing it is probably not a trivial endeavor. PRs welcome, though!
